subsys/net/lib/nrf_cloud/include/nrf_cloud_fota.h

```diff
@@ -102,19 +106,26 @@ typedef void (*nrf_cloud_fota_callback_t)
 typedef void (*nrf_cloud_fota_ble_callback_t)
 	(const struct nrf_cloud_fota_ble_job * const ble_job);
 
+struct nrf_cloud_fota_init_param {
+	nrf_cloud_fota_callback_t evt_cb;
+#if defined(CONFIG_NRF_CLOUD_FOTA_SMP)
+	dfu_target_reset_cb_t smp_reset_cb;
+#endif
+};
+
 /**
  * @brief Initialize the nRF Cloud FOTA module.
  *
  * @note This API must be called prior to using nRF Cloud FOTA and it must
  * return successfully.
  *
- * @param[in] cb FOTA event handler.
+ * @param[in] init Initialization parameters.
  *
  * @retval 0 If successful.
  *         1 If successful and a prior FOTA event has been completed.
  *           Otherwise, a (negative) error code is returned.
  */
-int nrf_cloud_fota_init(nrf_cloud_fota_callback_t cb);
+int nrf_cloud_fota_init(struct nrf_cloud_fota_init_param const *const init);
```

Comment:
I think they want us to list such changes in the migration guide for the upcoming release, FYI

Reply:
even for library "internal" APIs?

Reply:
oh, no, sorry -- I didn't notice this was the internal header